core/trino-main/src/main/java/io/trino/tracing/TracingMetadata.java

```diff
@@ -901,9 +901,7 @@ public Optional<TopNApplicationResult<TableHandle>> applyTopN(Session session, T
     public Optional<TableFunctionApplicationResult<TableHandle>> applyTableFunction(Session session, TableFunctionHandle handle)
     {
         Span span = startSpan("applyTableFunction")
-                .setAttribute(TrinoAttributes.CATALOG, handle.getCatalogHandle().getCatalogName())
-                .setAttribute(TrinoAttributes.SCHEMA, handle.getSchemaFunctionName().getSchemaName())
-                .setAttribute(TrinoAttributes.FUNCTION, handle.getSchemaFunctionName().getFunctionName());
```

Comment:
It can record handle.toString(), just like we do for TableHandles in other methods in this class.